components/navigation/communityItems.js

```diff
@@ -3,5 +3,7 @@ export default [
   { text: 'Github Organization', href: 'https://github.com/asyncapi', target: '_blank', description: 'Want to sneak in the code? Everything we do is open-sourced in our Github organization.' },
   { text: 'Slack Workspace', href: 'https://asyncapi.com/slack-invite', target: '_blank', description: `Need help? Want to share something? Join our Slack workspace. We're nice people :)` },
   { text: 'Contributing', href: 'https://github.com/asyncapi?type=source#-contribute-to-asyncapi', target: '_blank', description: `We are always welcoming and looking for contributions. If you are interested check out our contribution guide.` },
-  { text: 'Technical Steering Committee', href: '/community/tsc', description: 'Get to know what is a TSC member, how you can become one, and meet our current TSC members.' }
+  { text: 'Technical Steering Committee', href: '/community/tsc', description: 'Get to know what is a TSC member, how you can become one, and meet our current TSC members.' },
+  { text: 'Google Season of Docs (GSoD)', href: '/community/gsod', description: 'All technical writers are welcome to participate, regardless of tech background or years of experience!' },
```

Comment:
The addition of these 2 options in the dropdown of `Community` actually makes it quite responsive. On smaller screens, these last 2 options are not seen while opening the `Community` dropdown. We need to provide a scroll if you think these options are to be added necessarily. Here's the preview:

![Screenshot from 2022-04-01 23-50-46](https://user-images.githubusercontent.com/76521428/161320350-fc598fc8-cabf-4081-89f5-11ff09656f41.png)